agent/installer/registry_test.go

```diff
@@ -41,6 +41,9 @@ var _ = Describe("Byohost Installer Tests", func() {
 			Expect(func() { r.AddBundleInstaller("rhel", "1.24", dummy124) }).NotTo(Panic())
 			r.AddOsFilter("ubuntu.*", "ubuntu")
 			r.AddOsFilter("rhel.*", "rhel")
+			r.AddK8sFilter("1.22.*", "1.22")
```

Comment:
Question - why are we not using complete semver of the k8s version? like `v1.22.3` or `v1.23.5`